hrtime_migrate/migrate_sqlite.py

```diff
@@ -0,0 +1,170 @@
+                                    FOREIGN KEY (id_worker) REFERENCES Worker (id)
+                                );"""
+
+    sql_create_registration_table = """CREATE TABLE IF NOT EXISTS Registration (
+                                    id integer PRIMARY KEY,
+                                    datetime text,
+                                    direction text,
+                                    id_worker integer,
+                                    id_card integer,
+                                    FOREIGN KEY (id_worker) REFERENCES Worker (id),
+                                    FOREIGN KEY (id_card) REFERENCES Card (id)
+                                );"""
+    
+    # create a database connection
+    conn = create_connection(db_with_prepath)
+
+    # create tables
+    if conn is not None:
+        # create tables
+        create_table(conn, sql_create_worker_table)
+        create_table(conn, sql_create_card_table)
+        create_table(conn, sql_create_registration_table)
+    else:
+        print("Error! cannot create the database connection.")
+    
+    conn.commit()
+    #close the connection
+    conn.close()
+
+
+
+#############################################################
+def transform_json(filename, table_name_in_json):
+    """ transform data from json to database insert
+    :param list_of_dics: [ {a : aa, b : bb}, ... ]
+    :return: list of list of data for migration [ [a,b,c], ... ]
+    """
+    path = "{}{}".format(pre_path_json, filename)
+    if os.path.exists(path):
+        with open(path, 'r', encoding='utf-8') as json_file:
+            source_data = json.load(json_file)
+            list_of_dics = source_data[table_name_in_json]
+            migrate_datas = []
+            for dic_row in list_of_dics:
+                row_values = []
+                for k, v in dic_row.items(): 
+                    row_values.append(v)
+                migrate_datas.append(row_values)
+            return migrate_datas
+    
+    else: 
+        print("File not exist on path: {}".format(path))
+        return None
+
+
+def migrate_json(target):
+        
+        #create a database connection
+        conn = create_connection(db_with_prepath)
+
+        # create tables
+        if conn is not None:
+            # migrate datas
+            if target in ["card", "all"]:
+                migration_data = transform_json("db_card.json", "card")
+                if not migration_data == None:
+                    conn.executemany('INSERT INTO Card VALUES(?,?,?,?);', migration_data)
+
+            if target in ["user", "worker", "all"]:
+                migration_data = transform_json("db_user.json", "user")
+                if not migration_data == None:
+                    conn.executemany('INSERT INTO Worker VALUES(?,?,?,?,?,?,?);', migration_data)
+
+            if target in ["registration", "all"]:
+                migration_data = transform_json("db_registration.json", "registration")
```

Comment:
Jak skonczysz z powyższym , to proszę zauważ że do pełnego abstrachowania pliku importu ci brakuje dosłownie Tabeli do której się importuje danę, wić polecam do listy "column_order", "data" dodąć jeszcze `"import_to": <Table>` takie coś. I wtedy skrypt będzie zautomatyzowany.